### ISSUE TITLE
Error: Could not find or load main class Files

### ISSUE THREAD

**Issue by damongilmore** [forcedotcom/dataloader#503]
Data Loader 55.0.1 getting the following Error:

Data Loader requires Java JRE 11 or later. Checking if it is installed...
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
2022-06-29 12:31:38,397 INFO  [main] controller.Controller initLog (Controller.java:472) - The log has been initialized
Error: Could not find or load main class Files
Caused by: java.lang.ClassNotFoundException: Files

Using AdoptOpenJDK\jre-11.0.11.9-hotspot

Note: Version 53.0.1 works fine...

**Comment by damongilmore** [comment]
Removed the quotes and now I get the following: 
2022-07-06 14:47:03,860 INFO  [main] controller.Controller initLog (Controller.java:472) - The log has been initialized
Unable to find SWT jar for Windows 10 : x86

**Comment by ashitsalesforce** [comment]
Hi @damongilmore , you need to remove the quotes from 2 places in dataloader.bat:

```

    set JAVA_HOME=%DATALOADER_JAVA_HOME%
```

and
```
    PATH=%JAVA_HOME%\bin\;%PATH%;
```

If you still see the above error after verifying that you do not have quotes when setting `JAVA_HOME` or `PATH`, do you see `swtwin32_x86_64` subfolder in your data loader installation folder? Also, edit log-conf.xml to set the Root element's level to debug as follows:
```
                <Root level="debug">
                        <AppenderRef ref="fileAppender"/>
                        <AppenderRef ref="STDOUT"/>
                </Root>
```

Next, run data loader and provide the console output for further troubleshooting.

Thanks.

**Comment by ashitsalesforce** [comment]
Closing the issue assuming it is resolved with the provided workaround.

**Comment by ashitsalesforce** [comment]
Issue #518 seems identical to this one. It was resolved with a patch of data loader (v55.0.3) that is provided in a comment of issue #518.
